client/packages/electron/src/nativePrint.ts

```diff
@@ -0,0 +1,33 @@
+import { BrowserWindow } from 'electron';
+
+export const nativePrint = async (html: string) => {
```

Comment:
Converting to pdf seems like the only way to do this, i couldn't figure out another way 🤷